aries_cloudagent/holder/in_memory.py

```diff
@@ -0,0 +1,189 @@
+"""Basic in-memory storage implementation of VC holder interface."""
+
+from aries_cloudagent.storage.record import StorageRecord
+from typing import Mapping, Sequence
+
+from ..core.in_memory import InMemoryProfile
+
+from ..storage.in_memory import InMemoryStorage, InMemoryStorageSearch
+
+from .models.vc_record import VCRecord
+
+from .vc_holder import VCHolder, VCRecordSearch
+
+VC_CRED_RECORD_TYPE = "vc_cred"
+
+
+def load_credential(record: StorageRecord) -> VCRecord:
+    """Convert an in-memory stored record into a VC record."""
+    tags = {}
+    contexts = []
+    types = []
+    subject_ids = []
+    schema_id = None
```

Comment:
A credential can have multiple schemas. Probably not something we're going to use atm, but maybe good to allow it in the API